torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -121,8 +122,11 @@ void GpuLower::lower() {
   const auto lowered_exprs =
       LoopNestGenerator::loweredExprs(fusion_, fusion_->exprs());
 
+  // Insert read after write smem syncs
+  const auto raw_sync_exprs = insertRAWThreadSynchronization(lowered_exprs);
```

Comment:
we could use better names than insertRAWThreadSynchronization and insertWARThreadSynchronization..

btw, they should spell insertRawThreadSynchronization and insertWarThreadSynchronization if we want to keep them

Reply:
Renamed.